WebCryptoAPI/import_export/importKey_failures.js

```diff
@@ -16,12 +16,13 @@ function run_test(algorithmNames) {
 // success and failure modes, resulting in a very large number of tests
 // performed.
 
-
     var allTestVectors = [ // Parameters that should work for importKey / exportKey
         {name: "Ed25519", privateUsages: ["sign"], publicUsages: ["verify"]},
```

Comment:
I know that it's not a part of this patch, but don't we want to call Ed25519/Ed448 as EdDSA?

Reply:
Do you mean replacing the "Ed" prefix with EdDSA ?

Reply:
Ah, maybe in the end is not the best idea, as for Ecdsa we use "ECDSA" as an alg, but for "Ed25519", well, "Ed25519", not EdDSA